servlet-galleon-pack/src/main/resources/feature_groups/standalone-load-balancer.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature-group-spec name="standalone-load-balancer" xmlns="urn:jboss:galleon:feature-group:1.0">
+
+    <feature spec="socket-binding-group">
+        <param name="socket-binding-group" value="standard-sockets"/>
+        <param name="default-interface" value="public" />
+        <param name="port-offset" value="${jboss.socket.binding.port-offset:0}"/>
+        <feature-group name="management-sockets"/>
+        <feature-group name="load-balancer-sockets"/>
+    </feature>
+
+    <feature-group name="logging"/>
+    <feature-group name="io"/>
+
+    <feature-group name="security-realms">
+       <include feature-id="core-service.management.security-realm:core-service=management,security-realm=ManagementRealm">
+            <feature spec="core-service.management.security-realm.authentication.local">
+                <unset param="skip-group-loading"/>
+            </feature>
+       </include>
+       <include feature-id="core-service.management.security-realm:core-service=management,security-realm=ApplicationRealm">
+            <feature spec="core-service.management.security-realm.authentication.local">
+                <unset param="skip-group-loading"/>
```

Comment:
This is incorrect in the legacy file that this was added to match and will not be necessary when my fix for https://issues.jboss.org/browse/WFLY-10356 is merged.

Reply:
https://github.com/wildfly/wildfly/commit/59cd21c4f81b7e318fbd17b551cf85c85937ede8 fixes this.